core/src/main/scala/za/co/absa/spline/harvester/LineageHarvester.scala

```diff
@@ -122,7 +130,7 @@ class LineageHarvester(
         ).optionally[Duration]((m, d) => m + (ExecutionEventExtra.DurationNs -> d.toNanos), result.toOption)
 
         val ev = ExecutionEvent(
-          planId = plan.id,
+          planId = plan.id.get,
```

Comment:
As an exec plan ID is generated a few lines above can we reuse the value directly instead of unwrapping an `Option` here?
```suggestion
          planId = planId,
```